pkg/download/download_writer.go

```diff
@@ -53,21 +53,42 @@ func WriteToDisk(fs afero.Fs, writerContext WriterContext) error {
 }
 
 func writeToDisk(fs afero.Fs, writerContext WriterContext) error {
-
 	log.Debug("Preparing downloaded data for persisting")
 
-	manifestName := getManifestFilePath(fs, writerContext)
-	m := createManifest(writerContext)
+	manifestFilePath := getManifestFileName(fs, writerContext)
+	projectFolderPath := getProjectFolderName(fs, writerContext)
+
+	projectDefinition := manifest.ProjectDefinitionByProjectID{
```

Comment:
any deeper reason to pull the manifest creation out of its own named function?

Reply:
phu, no really **strong** argument.. I just didn't want to have the `getProjectFolderName()` call inside the `createManifest()` func and the other `getManifestFileName()` outside.... but both outside kind of outside consecutively. So you see immediately that the manifest file name and project folder name are kind of not static.

also `createManifest()` was basically just constructing that struct.. there is not much complexity to hide...

Reply:
makes sense :) Was really just wondering, not asking to put it back into it's own function